components/common-go/experiments/types.go

```diff
@@ -32,10 +37,23 @@ type Attributes struct {
 // If the environment contains CONFIGCAT_SDK_KEY value, it vill be used to construct a ConfigCat client.
 // Otherwise, it returns a client which always returns the default value. This client is used for Self-Hosted installations.
 func NewClient() Client {
+	gitpodHost := os.Getenv("GITPOD_HOST")
```

Comment:
@mustard-mh it is a internal secret of supervisor which is a subject to change, why don't pass it as an argument?